src/components/feed/NodeDetails/NodeDetails.tsx

```diff
@@ -221,8 +234,42 @@ const NodeDetails: React.FC<INodeProps> = ({ expandDrawer }) => {
             <Button onClick={downloadAllClick} icon={<FaDownload />}>
               Download Files
             </Button>
+            <Popover_patternfly
+                  id="node-details-popover"
+                  aria-label="Download command popover"
+                  position="bottom"
+                  headerContent={<div>chrs Download Files</div>}
+                  hasAutoWidth
+                  bodyContent={
+                    <div>
+                      <p>Copy and Paste the URL below into your chrs cli to download the files</p>
```

Comment:
The wording doesn't make much sense here, specifically "into your chrs cli" could be reworded. How about more simply:

> Run the command below in a terminal to download the files.

Reply:
Thanks, we have updated it.